Modules/Scripted/ScreenCapture/ScreenCapture.py

```diff
@@ -1050,10 +1068,13 @@ def captureImageFromView(self, view, filename, transparentBackground=False):
       imageClipper.Update()
       capturedImage = imageClipper.GetOutput()
 
-    writer = self.createImageWriter(filename)
-    writer.SetInputData(self.addWatermark(capturedImage))
-    writer.SetFileName(filename)
-    writer.Write()
+    if isinstance(volumeNode, slicer.vtkMRMLVectorVolumeNode):
```

Comment:
If volume node type is not correct (e.g., labelmap volume) then raise an exception.